ixdiagnose/artifacts/items/directory.py

```diff
@@ -0,0 +1,34 @@
+import functools
+import os
+import shutil
+
+from pathlib import Path
+from typing import Optional, Tuple
+
+from .base import Item
+
+
+def get_directory_size(directory: str) -> int:
+    return sum(i.lstat().st_size for i in Path(directory).rglob('*')) + Path(directory).stat().st_size
+
+
+def copy2(copied_files: list, src: str, dst: str) -> str:
+    copied_files.append(shutil.copy2(src, dst))
```

Comment:
I believe you want to copy source here instead of the destination keeping in line with what `File` item does ?